### ISSUE TITLE
Milestone 3 demo 28 March 2023

### ISSUE THREAD

**Issue by michielbdejong** [SURFnet/rd-sram-integration#125]
Preparing this in https://github.com/SURFnet/rd-sram-integration/issues/98 and https://github.com/SURFnet/rd-sram-integration/issues/96

**Comment by michielbdejong** [comment]
The demo is mostly working; some loose ends:

* [ ] #124 
* [ ] #126 
* [ ] #127
* [ ] #128

Most of these are minor, but I'm thinking maybe it's nicer to postpone the demo until after #127 is resolved; it will not make a difference for demonstrating the components Ponder Source delivers, but I think that will make the demo more appealing for SURF.

**Comment by michielbdejong** [comment]
The demo is tomorrow at 2pm
I've been debugging the setup a bit, looking good so far but have not got it fully working yet
I also still need to prepare the narrative and some slides
Just finished reinstalling all servers so they have a `SITE` env var inside the Docker image.
Will finish debugging tomorrow!
